14-11-2018/grzesiek/rot13/rot13.rb

```diff
@@ -0,0 +1,19 @@
+class String
+  ALPHABET = ("a".."z").to_a
+  ALPHABET_MOVED = ALPHABET.rotate(13).join
+  C_ALPHABET = ("A".."Z").to_a
+  C_ALPHABET_MOVED = C_ALPHABET.rotate(13).join
+
+  def rot13
+    return "String is empty" if self.empty?
+    raise ArgumentError.new("Wrong character in your string") if self.include? "ą"
```

Comment:
I like that you put it in one line but you would have to extend it to cover all other polish letters like ę for example